runtime/lua/vim/treesitter/query.lua

```diff
@@ -208,12 +208,34 @@ end
 ---@param opts table Optional parameters.
 ---          - concat: (boolean default true) Concatenate result in a string
 function M.get_node_text(node, source, opts)
-  opts = opts or {}
-  local concat = vim.F.if_nil(opts.concat, true)
 
   local start_row, start_col, start_byte = node:start()
   local end_row, end_col, end_byte = node:end_()
 
+  return M.get_range_text({start_row, start_col, start_byte, end_row, end_col, end_byte}, source, opts)
+end
+
+--- Gets the text corresponding to a given rangen
```

Comment:
should there be some (extremely brief) hint about the fact that a "range" maps to the input text (which is usually a buffer)? Or is that understood? Maybe we need a tag like `treesitter-range` to link to?